components/grid/grid.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>modal</title>
+    <link rel="stylesheet" href="/components/grid/grid.css" />
+    <link rel="stylesheet" href="/common.css" />
+    <link rel="stylesheet" href="/navbar.css" />
+    <link rel="stylesheet" href="/documentation.css" />
+    <link rel="stylesheet" href="/main.css" />
```

Comment:
just a suggestion: what you can do is : you can import  your navbar.css in main.css , So that you don not have to link it repeatedly  . and it will take th css from main.css

Reply:
thanks will try this.